playwright/register-form-validation.spec.ts

```diff
@@ -22,47 +55,41 @@ async function fillInitialsInput(page: Page, user: User): Promise<void> {
 
 async function fillEmailInput(page: Page, user: User): Promise<void> {
   const emailInput: Locator = page.getByPlaceholder('vilnaCRM@gmail.com');
-  await emailInput.fill('hello@sdf');
-  await expect(
-    page.getByText("Email must contain '@' and '.' symbols")
-  ).toBeVisible();
+  for (const expectation of expectations) {
```

Comment:
why you didn't use map or forEach?

Reply:
It was a comfortable way for me, should I change it?

Reply:
yeah, change it, it will be looks more clear

Reply:
foreach is not adapted to async/await. Its adaptation to async/await will not look better than "for of"
https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971

Reply:
okay, you can leave for of